projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/generated-methods-and-properties.migration.ts

```diff
@@ -438,6 +452,20 @@ export const GENERATED_METHODS_AND_PROPERTIES_MIGRATION: MethodPropertyDeprecati
       comment:
         "// TODO:Spartacus - Property 'activeCartService' was removed from Class 'NotCheckoutAuthGuard'. Use 'activeCartFacade' instead.",
     },
+    {
+      class: 'PaymentFormComponent',
+      importPath: '@spartacus/checkout/base/components',
+      deprecatedNode: 'modalService',
+      comment:
+        "// TODO:Spartacus - Property 'modalService' was removed from Class 'PaymentFormComponent'. Use 'launchDialogService' instead.",
```

Comment:
this is constructor change.

Reply:
This constructor change entry is already present in generated-constructor.migration.ts